src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -299,6 +300,19 @@ public void SetCheckpointDir(string directory)
             _jvmObject.Invoke("setCheckpointDir", directory);
         }
 
+        /// <summary>
+        /// Broadcast a read-only variable to the cluster, returning a Microsoft.Spark.Broadcast
+        /// object for reading it in distributed functions. The variable will be sent to each 
+        /// executor only once.
+        /// </summary>
+        /// <typeparam name="T">Type of the variable being broadcast</typeparam>
+        /// <param name="value">Value/variable to be broadcast</param>
+        /// <returns>A Microsoft.Spark.Broadcast object</returns>
```

Comment:
nit: let's just do `A Broadcast object` (cref is optional but you can add if you want)